lib/live_debugger_web/components.ex

```diff
@@ -72,6 +72,57 @@ defmodule LiveDebuggerWeb.Components do
     """
   end
 
+  @doc """
+  Renders an input with label.
+  """
+  attr(:id, :any, default: nil)
+  attr(:name, :any)
+  attr(:label, :string, default: nil)
+  attr(:value, :any)
+
+  attr(:type, :string,
+    default: "text",
+    values: ~w(checkbox color date datetime-local email file month number password
```

Comment:
Checkbox is already defined in the component above, also I'm not sure if this input will for each of these type. I'd consider removing allowed values from this field tbh